code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -312,7 +332,10 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 		/obj/item/dice/d20/fate/stealth/cursed = 1, //Only rolls 1
 		/obj/item/clothing/shoes/jackboots/fast = 1,
 		/obj/item/clothing/suit/armor/reactive/table = 1,
-		/obj/item/storage/box/donkpockets/donkpocketgondola = 1
+		/obj/item/storage/box/donkpockets/donkpocketgondola = 1,
+		/obj/item/instrument/saxophone/spectral = 1,
+		/obj/item/instrument/trumpet/spectral = 1,
+		/obj/item/instrument/trombone/spectral = 1,
```

Comment:
this will really bloat the list to be spectral instruments most of the time. make spectral instruments a weighted list with 1

Reply:
Yeah, do that.

Reply:
Fixed with 4d4051d